client/components/date-range/shortcuts.tsx

```diff
@@ -86,7 +90,7 @@ const DateRangePickerShortcuts = ( {
 };
 
 DateRangePickerShortcuts.propTypes = {
-	currentShortcut: PropTypes.string,
+	currentShortcutId: PropTypes.string,
```

Comment:
it's called `currentShortcut`, `appliedShortcut`, and `selectedShortcut` in different places, which we might want to align?

Reply:
I also struggled with the naming. For meaningfulness in different scopes, I would prefer to keep `selectedShortcut` and `appliedShortcut`. What do you think?

Reply:
sounds good 👍